packages/augur-ui/src/services/MeshTransferHandler.ts

```diff
@@ -0,0 +1,32 @@
+import * as Comlink from 'comlink';
+import { BigNumber } from '@0x/utils';
+import { mapValues } from 'lodash';
+
+const serialize = item => {
```

Comment:
Should be able to do this as part of JSON.serialize.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify

Reply:
I can't if I want to add extra information for deserialization.

Reply:
If we decide this those also support WS here then we should update zero validation login to handle "string-ed" big numbers.